README.md

```diff
@@ -35,6 +36,56 @@ $ swaggerhub configure
 ```
 
 Environment variables take precedence over the configuration file created by this command.
+
+## Additional configuration for SwaggerHub On-Premise
+
+If your SwaggerHub On-Premise instance uses a **self-signed or privately signed SSL certificate**, there are additional steps required to make SwaggerHub CLI trust this certificate.
```

Comment:
When talking about a specific or particular noun, you should use the definite article: the.

Ie. `there are additional steps required to make the SwaggerHub CLI trust this certificate.`